File: app/src/main/java/br/org/cesar/discordtime/stickysessions/logger/Logger.java
package br.org.cesar.discordtime.stickysessions.logger;

public interface Logger {
    void d(String tag, String message);
    void w(String tag, String message);
    void e(String tag, String message);
    void v(String tag, String message);
    void i(String tag, String message);
}

File: app/src/main/java/br/org/cesar/discordtime/stickysessions/logger/Logger.kt
package br.org.cesar.discordtime.stickysessions.logger;

interface Logger {
    fun d(tag: String, message: String)
    fun w(tag: String, message: String)
    fun w(tag: String, message: String, e: Throwable?)
    fun e(tag: String, message: String)
    fun e(tag: String, message: String, e: Throwable?)
    fun v(tag: String, message: String)
    fun i(tag: String, message: String)
}
